src/index.html

```diff
@@ -8,6 +8,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <main class="catalog">
+      <div class="card">
```

Comment:
These two tags are unnecessary, the task was to implement 6 blocks, you don't need to wrap them in any additional tags. This is probably causing problems with tests